### DISCUSSION TITLE
.Net8 上Docker引入非root用户运行，导致没有文件权限

### DISCUSSION THREAD

**Discussion by zhouyuguangsc** [dotnetcore/Natasha#243] [General]
在我使用.net 8并且docker使用非root用户时，会出现某个文件夹没有权限的问题，看文件夹名称应该是用于保存动态代码的，具体名称忘了大概是”Dynamicxxxxxx“这样。但是在我本地使用windows上的docker运行并不会发生问题

**Comment by zhouyuguangsc** [comment]
出现问题的环境是linux服务器加K8s

**Reply by NMSAzulX** [reply]
dockerfile 发一下看看

**Reply by zhouyuguangsc** [reply]
![1703219922254](https://github.com/dotnetcore/Natasha/assets/41465309/c90698b8-35a7-4d9b-974f-b9d67945afb7)
dockerfile就是用vs自动生成的，我现在是把用户那个注释掉，就没问题了
